tests/integration/test_numeric_array.py

```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+
+"""Integration-test suite for numeric-arrays."""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np
+
+from cr.cube.cube import Cube
+
+from ..fixtures import NA
+
+NAN = np.nan
```

Comment:
where is this COSTANT used? i don't see any usage in this file ... am I missing something?

Reply:
Good catch! It was used in the middle of the process but I ended up removing it. Fixed now.